src/set_theory/cardinal/basic.lean

```diff
@@ -716,6 +717,23 @@ begin
   exact mk_congr (equiv.ulift.trans $ equiv.Pi_congr_right $ λ i, equiv.ulift.symm)
 end
 
+lemma prod_of_fintype_eq {α : Type u} [fintype α] (f : α → cardinal.{v}) :
```

Comment:
Is the name better as `prod_eq_of_fintype` @eric-wieser ?

Reply:
No preference from me, whichever seems to best match the other `of_fintype`s in the rest of the file.